plugin.video.greenpeace/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.pyc
```

Comment:
Hi Pil65,

I'll would like to do it properly from my command line, but not sure i'm not breaking the "just one PR per commit" rule or (even worse) the whole local repo. I've deleted the ".gitignore" file from my local repo before by using "git rm .gitignore", but i cannot dare to do an --amend commit and re-push to my fork before reply this PR.

Please, if you can help me on this (i only use git from the command line) to do it in the right way i'll very appreciate it and i'll make it sure of avoid it for next PRs.

Many thanks in advance and best regards,
jamontes.